pkg/scheduler/framework/interface_test.go

```diff
@@ -87,6 +87,66 @@ func TestStatus(t *testing.T) {
 	}
 }
 
+func TestIsWaitStatus(t *testing.T) {
+	tests := []struct {
+		name            string
+		status          *Status
+		expectedCode    Code
+		expectedMessage string
+		expectedIsWait  bool
+		expectedAsError error
+	}{
+		{
```

Comment:
I think you'd better just add this case for `wait` in to TestStatus.

Reply:
So, let's merge them into the original `TestStatus` functions. Maybe refactor like this:
```
	isSuccess := func(s *Status) bool {
		return s.IsSuccess()
	}
	isWait := func(s *Status) bool {
		return s.IsWait()
	}

	tests := []struct {
		name            string
		status          *Status
		expectedCode    Code
		expectedMessage string
		statusFunc      func(*Status) bool
		expectedStatus  bool
		expectedAsError error
	}{
``` 

but it seems a little complex but more extensible. Glad to hear your advices @denkensk

Reply:
So, let's merge them into the original `TestStatus` functions. Maybe refactor like this:
```
	isSuccess := func(s *Status) bool {
		return s.IsSuccess()
	}
	isWait := func(s *Status) bool {
		return s.IsWait()
	}

	tests := []struct {
		name            string
		status          *Status
		expectedCode    Code
		expectedMessage string
		statusFunc      func(*Status) bool
		expectedStatus  bool
		expectedAsError error
	}{
``` 

but it seems a little complex but more extensible. Glad to hear your advices @denkensk